pyqt/pyqt5.json

```diff
@@ -0,0 +1,115 @@
+{
+    "name": "pyqt5",
+    "config-opts": [
+        "--concatenate=1",
+        "--confirm-license",
+        "--enable=QtCore",
+        "--enable=QtDBus",
+        "--enable=QtGui",
+        "--enable=QtNetwork",
+        "--enable=QtOpenGL",
+        "--enable=QtPrintSupport",
+        "--enable=QtQml",
+        "--enable=QtQuick",
+        "--enable=QtSql",
+        "--enable=QtTest",
+        "--enable=QtWebChannel",
+        "--enable=QtWidgets",
+        "--no-designer-plugin",
+        "--no-docstrings",
+        "--no-make",
+        "--no-qml-plugin",
+        "--no-tools",
+        "--api-dir=/app/share/qt/qsci/api/python",
+        "--qmake-setting=QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.8/'",
+        "--qmake-setting=QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.8/'",
```

Comment:
By default, it won't be built using the default compilation flags defined in the runtime. You'll need to rely on an external file as done in https://github.com/flathub/org.musicbrainz.Picard as flatpak doesn't expand environment viariables in `config-opts` AFAIK.